README.md

```diff
@@ -31,3 +31,12 @@ Launches the Cypress test runner. Cypress requires you to run `npm start` in par
 ```sh
 npm run cypress
 ```
+
+To get `holderChangePersonaldata.js`:
+
+```sh
+cd cypress/fixtures/
+ln -s ../../../testdata/b2bs/holderChangePersonaldata.json .
```

Comment:
As we do in webforms-api i would git add the relative link and just say we require the private repo to be cloned with that name. You will need the repo anyway but at least you remove a setup step.

Reply:
Ok! As we are working on tests in another branch, we leave it as technical debt if it is not done there. :+1: